frontend/src/pages/releases/link-preview.mdx

```diff
@@ -0,0 +1,246 @@
+
+Above all, it's all FREE, so we have more budget to spend on funsies:
+
+<img
+  src="/images/release_notes/link-preview/slack-saved-money.png"
+  alt='Slack post of Alex saying "now lets use that extra money we saved for our BARCELONA DINNER"'
+  width={600}
+/>
+
+As promised, we are including all these funny Slack exchanges in this post, just to show you what we have been through:
+
+<img
+  src="/images/release_notes/link-preview/slack-slack-message.png"
+  alt='Slack post of Alex, saying "all of these slack messages can be included in the blog when we finally get this"'
+  width={800}
+/>
+
+## Further improvements
+
+We don't plan to stop here. We have several improvements in this vein in our todo list:
+
+- Currently, the preview only contains plaintext, which is straightforward to retrieve. We want to dynamically create images to show up in the preview, similar to how the image preview for a tweet is the tweet itself. We still want to figure out exactly what that image should be—whether it's purely text-based, a collage of course-related images, or a combination of both—if you have any thoughts, [let us know](https://feedback.coursetable.com/feature-requests/p/social-media-preview-image)!
+- We want to serve _real_ HTML, rendered from our frontend code, instead of a fake HTML file that only contains head metadata. This may be important because we are soon allowing search engines to index each course, and the more information we have, the more discoverable the course pages are.
+- We also plan to further enhance our general handling of metadata to include information such as last modified ([#1619](https://github.com/coursetable/coursetable/issues/1619)), professor(s), related courses, and other relevant data, which is useful for search engine display.
+
+This experience has not only improved how CourseTable links appear on social media but also enables CourseTable courses to be indexed by search engines. For us, it provides valuable insights into our tech setup and experience for handling similar issues in the future. It is a great example of why we think CourseTable is so great. Through collaboration, creativity, and perseverance, we have made CourseTable more robust and user-friendly than ever before. We continue to fulfill our goal of making course selection and sharing as seamless and informative as possible for all Yale students—all while keeping it completely open and free.
```

Comment:
- indexed by search engines

a little redundant and not totally true yet wdyt

Reply:
It's true though! Without this, I don't think search engines can find those stuff because loading data takes time.